.github/workflows/tox.yml

```diff
@@ -0,0 +1,142 @@
+name: tox
+
+on:
+  create:  # is used for publishing to PyPI and TestPyPI
+    tags:  # any tag regardless of its name, no branches
+  push:  # only publishes pushes to the main branch to TestPyPI
+    branches:  # any branch but not tag
+      - >-
+        **
+    tags-ignore:
+      - >-
+        **
+  pull_request:
+  schedule:
+    - cron: 1 0 * * *  # Run daily at 0:01 UTC
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      fail-fast: false
+      matrix:
+        name: [
+          "linting",
```

Comment:
Should be "lint"

Reply:
In fact, this whole structure maybe should be pulled dynamically by a previous build step that generates the list from calling "tox -l"?